app/services/vot/legacy_component_values.rb

```diff
@@ -0,0 +1,84 @@
+module Vot
+  module LegacyComponentValues
+    ## Identity proofing ACR values
+    LOA1 = ComponentValue.new(
+      name: Saml::Idp::Constants::LOA1_AUTHN_CONTEXT_CLASSREF,
+      description: 'Legacy LOA1',
+      implied_component_values: [],
+      requirements: [],
+    )
+    LOA3 = ComponentValue.new(
+      name: Saml::Idp::Constants::LOA3_AUTHN_CONTEXT_CLASSREF,
+      description: 'Legacy LOA3',
+      implied_component_values: [],
+      requirements: [:aal2, :identity_proofing],
+    )
+    IAL1 = ComponentValue.new(
+      name: Saml::Idp::Constants::IAL1_AUTHN_CONTEXT_CLASSREF,
+      description: 'Legacy IAL1',
+      implied_component_values: [],
+      requirements: [],
+    )
+    IAL2 = ComponentValue.new(
+      name: Saml::Idp::Constants::IAL2_AUTHN_CONTEXT_CLASSREF,
+      description: 'Legacy IAL2',
+      implied_component_values: [],
+      requirements: [:aal2, :identity_proofing],
+    )
```

Comment:
We can set up AAL2 as an implied requirement right?
```suggestion
    IAL2 = ComponentValue.new(
      name: Saml::Idp::Constants::IAL2_AUTHN_CONTEXT_CLASSREF,
      description: 'Legacy IAL2',
      implied_component_values: [AAL2],
      requirements: [:identity_proofing],
    )
```

Reply:
We can though that will cause the "AAL2" ACR value to appear in the expanded form for this. That may be problematic since there are some duplicates of AAL2 e.g. "AAL2+HSPD12". I was going to leave the implied components off for ACR values to maintain the existing behavior there.

Reply:
ok sounds good